jupyter-on-gke/iap_module/variables.tf

```diff
@@ -49,4 +49,16 @@ variable "client" {
     secret = string
   })
   description = "oauth client"
+}
+
+variable "url_domain_addr" { 
```

Comment:
Probably also needs a readme update.
I think we need a separate PR that either documents all the vars/options in the readme or in a different .md file referenced in the readme.

Reply:
Yeah definitely, I have a task for myself to add a definitions/"meta" file.

Reply:
I added a note to the readme as well.